### PR TITLE
[Action] Fix a bug that upload_symbols_to_crashlytics fails in specific cases

### PR DESCRIPTION
**Pull request by chuganzy**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

`upload_symbols_to_crashlytics` fails in specific cases.

### Description

For instance, if the path contains spaces it fails..

### CODE REVIEW

**Review by joshdholtz — CHANGES_REQUESTED** [review]
Good find! Two things...
1) Is there an issue related to this bug that we can close?
2) Can you a test to verify this fixes the space issue?

**Comment by chuganzy** [comment]
1) No, I could not find it.
2) Sure! Will do it soon!

File: fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
       it "uploads dSYM files" do
         binary_path = './spec/fixtures/screenshots/screenshot1.png'
         dsym_path = './spec/fixtures/dSYM/Themoji.dSYM'
-        gsp_path = './spec/fixtures/plist/Info.plist'
+        gsp_path = './spec/fixtures/plist/With Space.plist'
```
Comment (chuganzy) [inline fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb]: This test already covers `shellescape` so just switched the path here.

File: fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb [hunk 2]
```diff
@@ -54,6 +54,17 @@
...
+      it "raises exception if given gsp_path is not found" do
+        gsp_path = './spec/fixtures/plist/_Not Exist_.plist'
+        expect do
+          Fastlane::FastFile.new.parse("lane :test do
+            upload_symbols_to_crashlytics(
+              gsp_path: 'fastlane/#{gsp_path}',
+              api_token: 'something123')
+          end").runner.execute(:test)
+        end.to raise_error(FastlaneCore::Interface::FastlaneError)
+      end
```
Comment (chuganzy) [inline fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb]: Added the test for the case that `plist` file does not exits.

**Comment by chuganzy** [comment]
@joshdholtz addressed

File: fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb (hunk 1 above)
Comment (joshdholtz) [reply fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb]: 🙌

**Review by joshdholtz — APPROVED** [review]
Woot! Thank you 😊

**Pull request merged**